src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -62,6 +72,11 @@ public BindingWorkflow(IHost host, ProjectInformation project)
 
         #region Workflow state
 
+        public List<Project> BindingProjects
```

Comment:
Should be set, yes. But these will be COM objects.. yuk. Need to provide a custom comparer or do we have one already somewhere. What about threading issues etc?